src/morphology/english/regexNoun.js

```diff
@@ -88,8 +88,31 @@ const hispanic = [
 	};
 } );
 
+const possessiveToBase = [
+	[ /['’]s$/i, "" ],
+	[ /(s)(['’]s?)$/i, "$1" ],
+].map( function( a ) {
```

Comment:
I think it would be good to use a more descriptive variable than `a` (`word` maybe?).

Reply:
Well, it's not really a `word`, but a regex-pair, a match and a substitute. Not sure how to call them together...